app/Providers/EventServiceProvider.php

```diff
@@ -34,9 +48,8 @@ public function boot(): void {
         parent::boot();
 
         // Dispatch Jobs from Events
-        Event::listen(fn(UserCheckedIn $event)
-            => PostStatusOnTwitter::dispatchIf($event->shouldPostOnTwitter, $event->status));
-        Event::listen(fn(UserCheckedIn $event)
-            => PostStatusOnMastodon::dispatchIf($event->shouldPostOnMastodon, $event->status, $event->shouldChain));
+        Event::listen(fn(UserCheckedIn $event) => PostStatusOnTwitter::dispatchIf($event->shouldPostOnTwitter, $event->status));
+        Event::listen(fn(UserCheckedIn $event) => PostStatusOnMastodon::dispatchIf($event->shouldPostOnMastodon, $event->status, $event->shouldChain));
+        Event::listen(fn(WebhookCallFailedEvent $event) => Log::error("Webhook call failed: " . json_encode($event)));
```

Comment:
Maybe put this in a context?

```suggestion
        Event::listen(fn(WebhookCallFailedEvent $event) => Log::error("Webhook call failed.", ['event' => json_encode($event)]));
```